storage/storage.hpp

```diff
@@ -88,6 +88,10 @@ class Storage
   // downloading maps to a special place but not for continue working with them from this place.
   string m_dataDir;
 
+  // A list of urls of servers for downloading maps. It's necessary for storage integration tests.
+  // For example {"http://eu1.mapswithme.com/direct/mac/"}.
```

Comment:
Please, don't show our private URLS even in comments. For now, we have a single public URL (new-search.mapswithme.com) which is an alias to eu1, and will be removed soon. I know, all URLS can be easily extracted by strings or something, but our current policy is to not share URLS of our servers. Please, contact @deathbaba for more details.